### DISCUSSION TITLE
Bug in firmware? Pausing cause printer to move up and print in the air.

### DISCUSSION THREAD

**Discussion by johncotten** [xblax/flashforge_ad5m_klipper_mod#96] [Q&A]
I suspect this is probably part of the firmware. Has anyone went to pause a print using the pause button in fluidd and it move to pause but then immediately return to the print and start printing at a higher layer height? I've seen this a couple times, but only sporadically. Normally that button pauses fine.

**Comment by xblax** [comment]
I have not observed it, but I rarely use the PAUSE function during printing. If you run into that issue again, maybe you can provide the Klipper logfile, it could have useful information. Would also be interesting if any errors show in the console.

**Reply by ncpadotcpl** [reply]
this is happening to me, not sure what's going on..
